<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleMaterialBuilder</class>
 <widget class="QWidget" name="SimpleMaterialBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Ubuntu Condensed&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="0">
    <widget class="QLabel" name="temp_label">
     <property name="text">
      <string>Temp (K)</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="material_fraction_label_3">
     <property name="text">
      <string>Material</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="press_label">
     <property name="text">
      <string>Pressure (atm)</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QPushButton" name="clear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QPushButton" name="add_material">
     <property name="text">
      <string>Add Material</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QTextEdit" name="materialList"/>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="material_pressure">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>State</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="material_density">
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="g4materiallist">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QPushButton" name="create_material">
     <property name="text">
      <string>Create Material</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="g4elementlist">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="material_state">
     <item>
      <property name="text">
       <string>Undefined state</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Liquid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gas</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="fraction_label">
     <property name="text">
      <string>Fraction</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="element_label">
     <property name="text">
      <string>Element</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="material_stp">
     <property name="text">
      <string>STP</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="mat_composition">
       <property name="text">
        <string>Material composition</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="elementalcomposition">
       <property name="text">
        <string>Element Composition</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Density (g/cm3)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="material_temp">
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QDoubleSpinBox" name="material_fraction">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="material_name">
     <property name="placeholderText">
      <string>name of material</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="totfrac_label">
     <property name="text">
      <string>Total fraction: 0.0000</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Material Name</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="number_of_atoms">
       <property name="text">
        <string>Number of atoms</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="frac_mass">
       <property name="text">
        <string>Frac. Mass</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
